packaging/os/apt.py

```diff
@@ -394,10 +401,15 @@ def install(m, pkgspec, cache, upgrade=False, default_release=None,
         else:
             autoremove = ''
 
+        if only_upgrade:
+            only_upgrade = '--only-upgrade'
+        else:
+            only_upgrade = ''
```

Comment:
Since apt's build-deps also supports this option (see https://gist.github.com/mika/a6bba81a2fe0753c0c8a for a demo) I'd vote for:

````
        if only_upgrade:
            cmd += ' --only-upgrade'
````

after the `if build_dep:` (line 409) and before the `if default_release` (line 414) instead.

(I'm not an official ansible developer but I just implemented the same feature and stumbled upon this PR only after implementing it myself, other than this comment my implementation looks identical, so just my 2 cents. :))